frontend/dropdown.js

```diff
@@ -0,0 +1,49 @@
+const searchBox = document.querySelector('.dropdown input');
+const dropdownMenu = document.querySelector('.dropdown-menu');
+
+searchBox.addEventListener('focus', e => {
+    dropdownMenu.classList.remove('hidden');
+});
+
+searchBox.addEventListener('focusout', e => {
+    dropdownMenu.classList.add('hidden')    
+});
+
+function searchBoxFunction(todos){
+    searchBox.addEventListener('keyup', e => {
+        const results = getResults(searchBox.value.toLowerCase().trim(), todos);
+        const childs = dropdownMenu.querySelectorAll('.row');
+        childs.forEach(child => {
+            dropdownMenu.removeChild(child)
+        });
+        if(results){
+            for(let result of results){
+                makeRow(result)
+            }
+        }
```

Comment:
You should always try to reduce the amount of indentation in the code.

```suggestion
        if(!results){
            return;
        }
        
        for(let result of results){
            makeRow(result)
        }
```